src/webui/www/private/views/preferences.html

```diff
@@ -313,6 +313,17 @@
             <input type="checkbox" id="upnp_checkbox" />
             <label for="upnp_checkbox">QBT_TR(Use UPnP / NAT-PMP port forwarding from my router)QBT_TR[CONTEXT=OptionsDialog]</label>
         </div>
+        <fieldset class="settings">
+            <legend>
+                <input type="checkbox" id="ssl_enabled_checkbox" onclick="qBittorrent.Preferences.updateSslSettings();" />
+                <label for="ssl_enabled_checkbox">QBT_TR(Enable SSL Torrents)QBT_TR[CONTEXT=OptionsDialog]</label>
+            </legend>
+            <div class="formRow">
+                <label for="ssl_port_value">QBT_TR(Port used for incoming SSL connections:)QBT_TR[CONTEXT=OptionsDialog]</label>
+                <input type="text" id="ssl_port_value" style="width: 4em;" title="QBT_TR(Set to 0 to let your system pick an unused port)QBT_TR[CONTEXT=OptionsDialog]" />
+                <button type="button" id="ssl_random_port_button" style="margin-left: 1em;" onclick="qBittorrent.Preferences.generateRandomSslPort();">QBT_TR(Random)QBT_TR[CONTEXT=OptionsDialog]</button>
```

Comment:
```suggestion
                <label for="sslPortValue">QBT_TR(Port used for incoming SSL connections:)QBT_TR[CONTEXT=OptionsDialog]</label>
                <input type="text" id="sslPortValue" style="width: 4em;" title="QBT_TR(Set to 0 to let your system pick an unused port)QBT_TR[CONTEXT=OptionsDialog]" />
                <button type="button" id="sslRandomPortButton" style="margin-left: 1em;" onclick="qBittorrent.Preferences.generateRandomSslPort();">QBT_TR(Random)QBT_TR[CONTEXT=OptionsDialog]</button>
```

Reply:
Are you sure about this one ? it seems to go against the current convention in the HTML file.

Reply:
Yes. For now, new code (variables) should use camel case unless it is transmitted over the wire (for example the preference data).